### DISCUSSION TITLE
record not found while trying to expand a row [Sep 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6516] [Discord Threads]
> 🙋‍♂️ **afsalmcqueen**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### can you share your project info?  
#  
> 👨‍🚀 **wingkwong**  
#### by expanding a row, you should get    
####    
#### ```   
#### api/v1/db/data/noco/<project_id>/<table_id>/<row_id>   
#### ```  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### not able to expand any row.  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### /api/v1/db/data/noco/p_gcht7dh17qylek/Api/GET%20/api/v./insuranceplans//benefits  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### bug report : https://github.com/nocodb/nocodb/issues/6202  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### not urlencoding rawid : GET%20/api/v./insuranceplans//benefits  
#  
> 👨‍🚀 **wingkwong**  
#### you are expanding a row in grid view right?  
#  
> 👨‍🚀 **wingkwong**  
#### insuranceplans s project name and benefits is table name?  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### <@495466997304262658> My table name is Api  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### GET /api/v./insuranceplans//benefits : this is the value in the primary key column  
#  
> 👨‍🚀 **wingkwong**  
#### like this ?  
![image0](https://cdn.discordapp.com/attachments/1143146921817559131/1143457713003708476/image.png?ex=65172fbb&is=6515de3b&hm=ecf165c24a65cd90270a70fb63f46f864a3af1d20d9289f8089f385fe75b97c5&)  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### <@495466997304262658> Yes  
#  
> 👨‍🚀 **wingkwong**  
#### looks fine on my side  
![image0](https://cdn.discordapp.com/attachments/1143146921817559131/1143458515298549800/image.png?ex=6517307a&is=6515defa&hm=91b3e3a523c6cf202bb98529f86fceb112c2fb9420c967103d8bd3b7a36d4c97&)  
#  
> 👨‍🚀 **wingkwong**  
#### looks like your row Id is `GET /api/v./insuranceplans//benefits `  
#  
> 👨‍🚀 **wingkwong**  
#### oh ok i got wt u mean now  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### <@495466997304262658> this have to be from primary key column  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### <@495466997304262658> WOW thanks for the immediate fix https://github.com/nocodb/nocodb/pull/6207  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### when can i expect this to release  
#  
> 👨‍🚀 **wingkwong**  
#### will be included in next release which is scheduled on thur  
#  
> 👨‍🚀 **wingkwong**  
#### if you need that in urgent, you can use this image   
####    
#### ```   
#### nocodb/nocodb-timely:-pr-6207-20230822-0850   
#### ```  
#  
> 🙋‍♂️ **afsalmcqueen**  
#### Thanks <@495466997304262658>  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1143146921817559131)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
